packages/apps/src/microsoft/teams/apps/app_process.py

```diff
@@ -96,6 +113,10 @@ async def _build_context(
             # User token not available
             pass
 
+        # Get or refresh the graph token before passing it to the context
+        # Use None for tenant_id to get default tenant token
+        graph_token = await self._get_or_refresh_graph_token(None)
```

Comment:
why None? Why not pass in the tenant id? Otherwise it breaks the point of getting the tenant specific tokens. To be able to call graph from a bot _for a particular tenant_, we need the tenant specific token. 

(Might be worth testing it too - lmk if you need a pointer on how to test this scenario)